vdslib/src/vespa/vdslib/distribution/distribution.cpp

```diff
@@ -443,7 +429,7 @@ Distribution::getIdealNodes(const NodeType& nodeType,
         // Create temporary place to hold results. Use double linked list
         // for cheap access to back(). Stuff in redundancy fake entries to
         // avoid needing to check size during iteration.
-        std::list<ScoredNode> tmpResults(groupRedundancy, ScoredNode(0, 0, 0));
+        std::vector<ScoredNode> tmpResults(groupRedundancy, ScoredNode(0, 0, 0));
```

Comment:
Consider updating comment that explicitly mentions linked list usage

Reply:
Comment updated, and some other related improvements.